zb-db/src/main/java/io/zeebe/db/impl/rocksdb/transaction/ZeebeTransactionDb.java

```diff
@@ -125,6 +126,12 @@ private static long getNativeHandle(final RocksObject object) {
   private final EnumMap<ColumnFamilyNames, Long> columnFamilyMap;
   private final Long2ObjectHashMap<ColumnFamilyHandle> handelToEnumMap;
 
+  private final ReadOptions prefixReadOptions;
+  private final ReadOptions simpleReadOptions;
```

Comment:
Please call this `defaultReadOptions`, it is just the default configuration and there is nothing simple about it